sdks/java/core/src/main/java/org/apache/beam/sdk/coders/Coder.java

```diff
@@ -57,9 +53,13 @@
  * inferred. See {@link KvCoder} for an example of a simple compound {@link Coder} that supports
  * automatic composition in the {@link CoderRegistry}.
  *
+ * <p>A {@link CoderProvider} can be automatically registered with the {@link CoderRegistry} by the
```

Comment:
A few lines up, the commentary about `CoderProvider` reads as though it were written for `CoderFactory`. Now I'd say that `CoderProvider` has a broader use, and anyhow maybe we should just link there and say "read about how coders are inferred at `CoderRegistry` and `CoderProvider`.